docs/slides/slides-introduction.md

```diff
@@ -0,0 +1,246 @@
+duration: 20min
+routerMode: hash
+---
+
+# Welcome to Metaxy
+
+Accelerating AI experimentation with smart metadata handling without massive cost.
+
+<div @click="$slidev.nav.next" class="mt-12 py-1" hover:bg="white op-10">
+  Bridging BI reliability with AI velocity <carbon:arrow-right />
+</div>
+
+<div class="abs-br m-6 text-xl">
+  <button @click="$slidev.nav.openInEditor()" title="Open in Editor" class="slidev-icon-btn">
+    <carbon:edit />
+  </button>
+  <a href="https://github.com/anam-org/metaxy/tree/main/docs/slides/slides-introduction.md" target="_blank" class="slidev-icon-btn">
+    <carbon:logo-github />
+  </a>
+</div>
+
+<!--
+The last comment block of each slide will be treated as slide notes. It will be visible and editable in Presenter Mode along with the slide. [Read more in the docs](https://sli.dev/guide/syntax.html#notes)
+-->
+
+---
+layout: image-right
+image: /img/coffee.jpg
+transition: fade-out
+---
+
+# Same beans, different brews
+
+> In BI we run daily, idempotent, SQL-centric jobs; in AI we might launch 200 hyper-parameter sweeps per hour.
+
+<br>
+
+- **BI comfort**: curated dashboards and once-a-day refreshes
+- **AI reality**: hundreds of experiments, each with bespoke compute footprints
+
+<br>
+
+> **Metaxy** stitches these rhythms together so both sides sip consistent data with minimal cost but rapid experimentation
+
+<style>
+h1 {
+  background-color: #2B90B6;
+  background-image: linear-gradient(45deg, #4EC5D4 10%, #146b8c 20%);
+  background-size: 100%;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  -moz-text-fill-color: transparent;
+}
+</style>
+
+---
+layout: two-cols
+---
+
+## BI data pipelines
+
+- Predictable triggers and SLA windows
+- Centralized SQL engines own execution
+- Deterministic outputs feed compliance workflows
+
+::right::
+
+## AI data pipelines
+
+<v-clicks>
+
+- Experimentation across many model variants simultaneously
+- Massive, bursty compute outside the data warehouse
+- Flexible execution fabrics (GPUs, Ray clusters, notebooks, HPC)
+- Non-determinism from stochastic training runs
+
+</v-clicks>
+
+---
+
+# What Metaxy guarantees
+
+<v-clicks>
+
+- **Rapid experimentation** – ship new feature ideas without rewiring lineage
+- **Resource conservation** – gate GPU jobs behind metadata diffs
+- **Consistency** – trace every artifact back to its upstream inputs
+
+</v-clicks>
+
+<div mt-8 text-muted>
+Metaxy brings BI-like observability to the chaos of model iteration.
+</div>
+
+---
+
+# Only recompute what matters
+
+- Track feature versions and upstream dependencies automatically
+- Transitively recompute only the features whose inputs changed
+- Leave untouched metadata in place to save cost
+- Surface the minimal change set to orchestrators for targeted runs and integration
+
+---
+
+# Metaxy in action
+
+````md magic-move {lines: true}
+```py {1-3|1-5}
+from metaxy import init_metaxy
```

Comment:
refine coding example

Reply:
ideally show a dagster and ray integration as well once these are built

Reply:
also create an HPC example/reference https://github.com/ascii-supply-networks/dagster-slurm/ https://github.com/orgs/ascii-supply-networks/projects/4?pane=issue&itemId=132672627&issue=ascii-supply-networks%7Cdagster-slurm%7C35